superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -432,6 +433,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
 
   // Chart metadata
   const [chartVizType, setChartVizType] = useState<string>('');
+  const [selectedTabs, setSelectedTabs] = useState<string[] | null>(null);
```

Comment:
would it be easier for selectedTabs to be empty array when nothing is selected instead of null? If we need to pass `null` to the backend, we could always put that logic in the api call instead of needing to worry about it in our components